threatstream_connector.py

```diff
@@ -2053,11 +2060,7 @@ def _save_pcap_to_vault(self, response, container_id, action_result):
 
         # Creating temporary directory and file
         try:
-            if hasattr(Vault, 'get_vault_tmp_dir'):
-                temp_dir = Vault.get_vault_tmp_dir()
-            else:
-                temp_dir = "/opt/phantom/vault/tmp/"
-            temp_dir = temp_dir + '/{}'.format(uuid.uuid4())
+            temp_dir = Vault.get_vault_tmp_dir() + f"/{uuid.uuid4()}"
```

Comment:
I recommend using os.path.join for such cases to make sure that the path won't have e.g. double '/' inside.
```suggestion
            temp_dir = os.path.join(Vault.get_vault_tmp_dir(), f"{uuid.uuid4()}")
```
Just the suggestion tho